### ISSUE TITLE
Deletion definition nomenclature may be wrong

### ISSUE THREAD

**Issue by theosanderson** [outbreak-info/outbreak.info#368]
Thanks for outbreak.info!

I was trying to find a report for `B.1.617.2` with `ΔY144`.

If you begin entering the deletion you are instructed to `Specify a deletion like "S:DEL69/70"`

However, that specific example, `DEL69/60` refers to a deletion of residues 69 AND 70. I.e. a double deletion. Therefore my expectation would be for a single deletion I would enter something like `S:DEL144`. However that doesn't work. And `S:DEL144/144` is accepted, but then returns no sequences. However `S:DEL144/145` yields what I think is the result I wanted:  `ΔY144`.

However I think this is mis-specified. I think this means that when people enter `S:DEL69/70` results will be returned which include actually just the deletion at position 69 and not the deletion at position 70.

Apologies if I have got anything wrong.

**Comment by theosanderson** [comment]
(I guess "wrong" here means "inconsistent with how most people use del69/70")

**Comment by flaneuse** [comment]
Thanks @theosanderson for flagging this.  This is a known issue that we're working on. Right now, deletions are specified in chunks, so DEL144/145 will return any sequence with a deletion spanning residue 144-145. Unfortunately, due to the way we're storing the data, that means that if you search for DEL144 alone, any sequence that contains both the 144 and 145 deletions are not included.  We're in the process of splitting up the deletions so that you can search for DEL144 (grabbing any sequence with this mutation regardless if it's a single amino acid or longer), DEL145, or DEL144 **AND** DEL145.

DEL69/70 should include both deletions at 69 and 70, not just 69.

The DEL144/144 vs. DEL144 is another [issue that we're fixing](https://github.com/outbreak-info/outbreak.api/issues/13). The report you want is at https://outbreak.info/situation-reports?pango=B.1.617.2&muts=S%3ADEL144 ([or all DEL144])(https://outbreak.info/situation-reports?muts=S%3ADEL144), but we need to change the input options so either DEL144 or DEL144/144 is accepted. **Note though that this report only contains sequences that _only_ have DEL144, not ones that have DEL144/145, as described above.**  @gkarthik can we add this change to the updates this week, so at least single amino acid deletions are specified correctly?

 Hope this helps and we'll keep you updated on fixing these bugs.

**Comment by flaneuse** [comment]
Yeah, sorry!  That was a bad decision on our part.  That fix should be up by the end of the week.

We'll work on splitting of deletions into individual components next week.
